base_station/gps/src/__main__.py

```diff
@@ -0,0 +1,51 @@
+import serial
+import time
+import json
+from os import getenv
+from rover_common import aiolcm
+from rover_msgs import RTCM
+
+
+class GPS_Base_Station():
+
```

Comment:
NIT: Remove?

Reply:
Could you be more clear as to what you want me to remove? Is it the white space or the class definition? The white space is following PEP8 format due to our linter, and I don't see an issue with using a class to provide additional functionality.

Reply:
I was talking about the whitespace. I attempted to comment on just the one line but it doesn't seem like that was made clear.